src/backend/marsha/core/templates/core/lti_video.html

```diff
@@ -3,6 +3,9 @@
 <!DOCTYPE html>
 <html>
   <head>
+    {% if environment in authorized_environment%}
```

Comment:
why do you choose to call it "authorized environments"? Have you tried just checking that cloudfrontè_domain is set?

Reply:
it is always set, or we should set it only if in `production`, `staging`, etc